### PR TITLE
Added Appium RSpec and Cucumber examples

### PR DESCRIPTION
**Pull request by spider-sauce**

### CODE REVIEW

File: appium-examples/Cucumber/emusim/Rakefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+def run_tests(deviceName, platformName, platformVersion, app, junit)
+  system("deviceName=\"#{deviceName}\" platformName=\"#{platformName}\" platformVersion=\"#{platformVersion}\" app=\"#{app}\" parallel_cucumber features -n 20")
```
Comment (titusfortner) [inline appium-examples/Cucumber/emusim/Rakefile]: you... shouldn't need the quotes, these are setting environment variables, which... isn't the ideal way to do this either, but will work for now. :)
```
system "deviceName=#{deviceName} platformName=#{platformName} platformVersion=#{platformVersion} app=#{app} parallel_cucumber features -n 20"
```

File: appium-examples/Cucumber/emusim/features/step_definitions/comment_steps.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Given /^I click on the comment box$/ do
+  commentInput = @driver.find_element(:id, "comments")
```
Comment (titusfortner) [inline appium-examples/Cucumber/emusim/features/step_definitions/comment_steps.rb]: Ruby variables are snake case not camel case

File: appium-examples/Cucumber/emusim/features/support/env.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+require 'appium_lib'
+require 'sauce_whisk'
+require 'rspec'
+
+Before do | scenario |
+  # need to configure env variables for browser
+  caps = {
+      caps: {
+          platformVersion: "#{ENV['platformVersion']}",
```
Comment (titusfortner) [inline appium-examples/Cucumber/emusim/features/support/env.rb]: Ruby should convert snake case to camel case for these
Also, don't need quotes for these:
```
platformVersion: ENV['platformVersion']
```

File: appium-examples/Cucumber/emusim/Rakefile (hunk 1 above)
Comment (spider-sauce) [reply appium-examples/Cucumber/emusim/Rakefile]: sounds good.

File: appium-examples/Cucumber/emusim/features/support/env.rb (hunk 3 above)
Comment (spider-sauce) [reply appium-examples/Cucumber/emusim/features/support/env.rb]: fixed

File: appium-examples/Cucumber/emusim/features/step_definitions/comment_steps.rb (hunk 2 above)
Comment (spider-sauce) [reply appium-examples/Cucumber/emusim/features/step_definitions/comment_steps.rb]: changed

**Pull request merged**